### PR TITLE
Renames Nervousness to Stuttering

### PR DESCRIPTION
**Pull request by VTCobaltblood**
Title. IMO it makes more sense, discourages people playing nervous characters to always pick stuttering for no reason, and gives more freedom in creating characters (speech impediment without a stereotypical trait).
Nothing except for the name should be affected.

### CODE REVIEW

**Comment by VTCobaltblood** [comment]
frick...

File: code/game/dna/genes/disabilities.dm [hunk 1]
```diff
@@ -93,8 +93,8 @@
 		block=TWITCHBLOCK
 
 /datum/dna/gene/disability/nervousness
```
Comment (Arrow768) [inline code/game/dna/genes/disabilities.dm]: Change the names in the paths aswell and all references to it.

File: code/modules/mob/abstract/new_player/character_traits.dm [hunk 2]
```diff
@@ -15,8 +15,8 @@
 	H.disabilities |= NEARSIGHTED
 
 /datum/character_disabilities/nervous
```
Comment (Arrow768) [inline code/modules/mob/abstract/new_player/character_traits.dm]: And here aswell

File: code/game/dna/genes/disabilities.dm (hunk 1 above)
Comment (VTCobaltblood) [reply code/game/dna/genes/disabilities.dm]: Is this really needed? I deliberately avoided this to make as less of an impact as possible.

Comment (Arrow768) [reply code/game/dna/genes/disabilities.dm]: Yes, that is needed to avoid confusions in the future.

Comment (VTCobaltblood) [reply code/game/dna/genes/disabilities.dm]: Done now, hasn't caused runtime issues in testing

**Pull request merged**